build.gradle

```diff
@@ -0,0 +1,53 @@
+apply plugin: 'java'
+apply plugin: 'application'
+apply plugin: 'pmd'
+
+group 'com.github'
```

Comment:
better to start with correct group - for example - you nick

Reply:
Do you mean `com.github.aravij`?

Reply:
Fixed.